client/src/component/listHeader/listHeader.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
```

Comment:
Should be fine but in the future, make sure to always branch off of `main`. 

Here you branched off of your old branch where you created the `listHeader` and that is why these files show up as part of your PR.